nixos/modules/services/logging/logstash.nix

```diff
@@ -88,7 +88,7 @@ in
         example = ''
           # Read from journal
           pipe {
-            command => "''${pkgs.systemd}/bin/journalctl -f -o json"
+            command => "${pkgs.systemd}/bin/journalctl -f -o json"
```

Comment:
I think this may have been intentional, to avoid having to evaluate the package.